### ISSUE TITLE
multi line comments not "sticky"

### ISSUE THREAD

**Issue by timgraham** [timothycrosley/isort#246]
With 3.9.5...
```
from django import forms
# While this couples the geographic forms to the GEOS library,
# it decouples from database (by not importing SpatialBackend).
from django.contrib.gis.geos import GEOSException, GEOSGeometry
```
is transformed to:
````
from django import forms
# it decouples from database (by not importing SpatialBackend).
from django.contrib.gis.geos import GEOSException, GEOSGeometry
# While this couples the geographic forms to the GEOS library,
from django.utils.translation import ugettext_lazy as _
```

**Comment by timothycrosley** [comment]
Thanks for pointing out this issue! I'll be working to have a fix ready for the next release.

~Timothy

**Comment by timothycrosley** [comment]
This is now fixed in release 3.9.6.

Thanks again!

~Timothy
